examples/gaussian-ssm/script.jl

```diff
@@ -1,13 +1,72 @@
 # # Gaussian State Space Model
 using AdvancedPS
-using AbstractMCMC
+using Random
 using Distributions
-using Libtask: produce
-using StatsFuns
-
 using Plots
 
-x = [1, 2, 3]
-y = [2, 5, 1]
+# Parameters
+θ = @NamedTuple begin
```

Comment:
Maybe consider a more explicit variable name:

```suggestion
Parameters = @NamedTuple begin
```